diff --git a/node_modules/wikibase-edit/lib/request/login.js b/node_modules/wikibase-edit/lib/request/login.js
index 7bae86f..0856a35 100644
--- a/node_modules/wikibase-edit/lib/request/login.js
+++ b/node_modules/wikibase-edit/lib/request/login.js
@@ -45,14 +45,14 @@ const getSessionCookies = (loginUrl, config, headers) => async ({ cookies, token
     throw error_.new('failed to login: invalid username/password')
   }
 
-  const resCookies = res.headers.get('set-cookie')
+  const resCookies = res.headers.get('set-cookie') || ''
 
   if (!resCookies) {
-    throw error_.new('login error', res.statusCode, { body: resBody })
+    // throw error_.new('login error', res.statusCode, { body: resBody })
   }
 
   if (!sessionCookiePattern.test(resCookies)) {
-    throw error_.new('invalid login cookies', { cookies: resCookies })
+    // throw error_.new('invalid login cookies', { cookies: resCookies })
   }
 
   return parseSessionCookies(resCookies)
